### ISSUE TITLE
download template as tab separated txt file

### ISSUE THREAD

**Issue by mkuzak** [andi-nl/ANDI-frontend#172]
Currently the downloaded file is csv file.

**Comment by jvdzwaan** [comment]
In open office, the file becomes a comma separated file if you do ctrl-s. Is this also a problem if you use excel? (Just checking)

**Comment by jvdzwaan** [comment]
Wait, this is why it should be a txt file, and not a csv...

**Comment by jvdzwaan** [comment]
txt file is also automatically transformed to comma separated file in libreoffice.

**Comment by ridderl** [comment]
When you "upload data" the file selector filters on Microsoft excel .csv files, which excludes the .txt file that I just downloaded. Is it possible to set the file selector to "all files"?

**Comment by mkuzak** [comment]
Yes, it’s only changing one argument. I already discussed it with Janneke.


On 2 December 2016 at 18:52:24, ridderl (notifications@github.com) wrote:

When you "upload data" the file selector filters on Microsoft excel .csv
files, which excludes the .txt file that I just downloaded. Is it possible
to set the file selector to "all files"?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/andi-nl/ANDI-frontend/issues/172#issuecomment-264517308>,
or mute the thread
